docs/introduction/cron.md

```diff
@@ -31,8 +31,13 @@ The instance of cron is actually a named group of cron jobs.
 
 You can learn how to set up multiple cron instances in [Working with Multiple Cron Instances](../cookbook/working-with-multiple-cron-instances.md) cookbook.
 
+!!! note
+    By default it is not possible to run all crons at once. 
```

Comment:
I would rewrite the whole note :innocent: 
```
There is a special phing target `run-all-crons-serially` that (as the name suggests) allows you to run all the CRON modules serially. This is rather for testing reasons (we use it on our CI to check that no CRON is failing) and there is no point in using this target in production.
```

Reply:
Changed to something more straightforward